File: E-Shop/Front-End/ProjectFront/src/app/models/Product/EditProductDto.ts
import { Guid } from "guid-typescript";
import { ProductTypes } from "src/app/enums/ProductType";

export class EditProductDto{
    productId!: Guid;
    name!: string;
    price!: number;
    discount!: number;
    stock!: number;
    productType!: ProductTypes;
}

File: E-Shop/Back-End/Project.Data/Dto/Product/EditProductDto.cs
using Microsoft.AspNetCore.Http;
using Project.Data.Enities;

namespace Project.Data.Dto.Product;
public class EditProductDto {
    public Guid ProductId { get; set; }
    public string Name { get; set; }
    public double Price { get; set; }
    public int Discount { get; set; }
    public int Stock { get; set; }
    public ProductTypes ProductType { get; set; }
}
